src/serverdlg.cpp

```diff
@@ -72,17 +72,17 @@ CServerDlg::CServerDlg ( CServer* pNServP, CServerSettings* pNSetP, const bool b
                                          "registration failed, please choose another server list." ) );
 
     // custom directory server address
-    QString strCentrServAddr = "<b>" + tr ( "Custom Directory Server Address" ) + ":</b> " +
-                               tr ( "The custom directory server address is the IP address or URL of the directory "
-                                    "server at which the server list of the connection dialog is managed." );
+    QString strCustomDirectoryAddress = "<b>" + tr ( "Custom Directory Server Address" ) + ":</b> " +
+                                        tr ( "The custom directory server address is the IP address or URL of the directory "
+                                             "server at which the server list of the connection dialog is managed." );
 
-    lblCentralServerAddress->setWhatsThis ( strCentrServAddr );
-    edtCentralServerAddress->setWhatsThis ( strCentrServAddr );
-    edtCentralServerAddress->setAccessibleName ( tr ( "Directory server address line edit" ) );
+    lblDirectoryAddress->setWhatsThis ( strCustomDirectoryAddress );
+    edtDirectoryAddress->setWhatsThis ( strCustomDirectoryAddress );
+    edtDirectoryAddress->setAccessibleName ( tr ( "Directory server address line edit" ) );
 
-    cbxCentServAddrType->setWhatsThis ( "<b>" + tr ( "Server List Selection" ) + ":</b> " +
-                                        tr ( "Selects the server list (i.e. directory server address) in which your server will be added." ) );
-    cbxCentServAddrType->setAccessibleName ( tr ( "Server list selection combo box" ) );
+    cbxDirectoryType->setWhatsThis ( "<b>" + tr ( "Server List Selection" ) + ":</b> " +
+                                     tr ( "Selects the server list (i.e. directory server address) in which your server will be added." ) );
```

Comment:
Suggest "Selects the Directory Server list to which your server will be added."?

Reply:
See https://github.com/pljones/jamulus/commit/a83460052ce996c9c219e7548fcb233159d7df90

Reply:
(That commit id is now stale)